src/MallocBuffer.hpp

```diff
@@ -59,20 +59,20 @@ class MallocBuffer : public bufferManipulation::VoidBuffer
    */
   LVARRAY_HOST_DEVICE RAJA_INLINE constexpr
   MallocBuffer( bool=true ):
-    m_capacity( 0 ),
-    m_data( nullptr )
+    m_data( nullptr ),
```

Comment:
is seems like you are just changing the order here? Does this change behavior?

Reply:
No, but it's probably good policy to have the pointer as the first member. Then effectively to index into the array the compiler does `reinterpret_cast<T*>(*this)[ i ]` instead of `reinterpret_cast<T*>(*(this + member_offset))[ i ]`. It almost certainly doesn't make a difference but it won't hurt.